src/dune_pkg/dune

```diff
@@ -21,6 +21,7 @@
   build_info
   zeroinstall_solver
   fmt
-  xdg)
+  xdg
+  sha)
```

Comment:
Careful, the `sha` package is only vendored because of OPAM, I would be a bit hesitant to just use it directly in other code and probably go for `Digest` instead.

Or at least decide what kinds of hashes we want to use in Dune and deliberately vendor that hash library.

Reply:
switched to use `Digest`